TritonSE/LAP-Student-Tracker
# Feature/allisonbhavsar/28 image api

### Administrative Info

Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Github Issue]-[3-4 word description separated by dashes]`.

### What issue(s) does this branch close?
#28 

### Changes

What changes did you make?

- Created images table
- Altered users table to reference images table for profile pictures
- Created image API GET and PATCH routes

### Testing

How did you confirm your changes worked?

- Create new user and get user's <img_id>
`const res = await fetch('https://pngimg.com/uploads/mario/mario_PNG53.png').then(r => r.blob());  
const img = new Uint8Array(await res.arrayBuffer());`
- Call image PATCH route with <img_id> and above img variable
- Open http://localhost:3000/api/images/<img_id>
- Should render image from 'https://pngimg.com/uploads/mario/mario_PNG53.png'

### Confirmation of Change

Followed above steps in testing ^
